pandas/core/groupby/groupby.py

```diff
@@ -2338,11 +2339,11 @@ def head(self, n=5):
 
         >>> df = pd.DataFrame([[1, 2], [1, 4], [5, 6]],
         ...                   columns=['A', 'B'])
-        >>> df.groupby('A', as_index=False).head(1)
+        >>> df.groupby('A', as_index=True).head(1)
```

Comment:
```suggestion
        >>> df.groupby('A').head(1)
```

Let's leave it out since it has no effect? (and if doing so, also rneed emove the example below)

I don't think we necessarily need to show it has no effect, if we already stated it above, that doesn't seem the best focus of the example to me.

Reply:
fixed in latest commit